src/egon/data/datasets/osm/__init__.py

```diff
@@ -114,7 +120,9 @@ def to_postgres(num_processes=1, cache_size=4096):
 
 
 def add_metadata():
-    """Writes metadata JSON string into table comment."""
+    """Writes metadata JSON string into table comment.
```

Comment:
In my case the osm-Dataset was skipped because the version already existed. I started a clean run in an empty database now to make sure it is executed, but I thought that the Dataset-version should be always updated when the code changed. Or am I mistaken?

Reply:
\*cough\* yes, I forgot that, thanks.